<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1168</width>
    <height>752</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>C++ML Editor</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #7f7f7f;
margin-top: 0px;
margin-right: 0px;
margin-bottom: 0px;
margin-left: 0px;
spacing: 0px;
padding: 0px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
         </property>
         <property name="tabPosition">
          <enum>QTabWidget::North</enum>
         </property>
         <property name="tabShape">
          <enum>QTabWidget::Rounded</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="elideMode">
          <enum>Qt::ElideRight</enum>
         </property>
         <property name="usesScrollButtons">
          <bool>true</bool>
         </property>
         <property name="documentMode">
          <bool>false</bool>
         </property>
         <property name="tabsClosable">
          <bool>true</bool>
         </property>
         <property name="movable">
          <bool>false</bool>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Page</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="margin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QGraphicsView" name="graphicsView">
              <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
              </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Page</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1168</width>
     <height>27</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOPe"/>
    <addaction name="actionOpen_As"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="actionClose_All"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuPreferences">
     <property name="title">
      <string>Preferences</string>
     </property>
     <addaction name="actionGeneral"/>
     <addaction name="separator"/>
     <addaction name="actionInterface"/>
     <addaction name="actionShotcuts"/>
     <addaction name="actionAutocomplete"/>
     <addaction name="actionPlug_in"/>
    </widget>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="menuPreferences"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionTools"/>
    <addaction name="actionHistory"/>
    <addaction name="actionFiles"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout_C_Ml"/>
    <addaction name="actionC_Website"/>
   </widget>
   <widget class="QMenu" name="menuInsert">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuInheritance">
     <property name="title">
      <string>Inheritance</string>
     </property>
     <addaction name="actionPublic"/>
     <addaction name="actionProtected"/>
     <addaction name="actionPrivate"/>
    </widget>
    <addaction name="actionClass"/>
    <addaction name="actionStruct"/>
    <addaction name="actionUnion"/>
    <addaction name="actionTypedef"/>
    <addaction name="actionEnum"/>
    <addaction name="actionNamespace"/>
    <addaction name="separator"/>
    <addaction name="menuInheritance"/>
    <addaction name="actionComposition"/>
    <addaction name="actionAggregation"/>
    <addaction name="actionDependency"/>
    <addaction name="separator"/>
    <addaction name="actionFreeFunction"/>
    <addaction name="separator"/>
    <addaction name="actionTemplate"/>
    <addaction name="separator"/>
    <addaction name="actionLibrary"/>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuZoom">
     <property name="title">
      <string>Zoom</string>
     </property>
     <addaction name="action200"/>
     <addaction name="action175"/>
     <addaction name="action150"/>
     <addaction name="action125"/>
     <addaction name="action100"/>
     <addaction name="action85"/>
     <addaction name="action50"/>
     <addaction name="action25"/>
    </widget>
    <addaction name="actionZoom"/>
    <addaction name="actionZoom_2"/>
    <addaction name="menuZoom"/>
    <addaction name="actionFit_On_Screen"/>
    <addaction name="actionFullscreen"/>
    <addaction name="separator"/>
    <addaction name="actionAfficher_la_grille"/>
    <addaction name="actionMagnetic_Grid"/>
    <addaction name="separator"/>
    <addaction name="actionScreenshot"/>
    <addaction name="actionActualize"/>
    <addaction name="actionSelf_organization"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEdit"/>
   <addaction name="menuDisplay"/>
   <addaction name="menuInsert"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New...</string>
   </property>
  </action>
  <action name="actionOPe">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionOpen_As">
   <property name="text">
    <string>Open As...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionClose_All">
   <property name="text">
    <string>Close All</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionGeneral">
   <property name="text">
    <string>General</string>
   </property>
  </action>
  <action name="actionInterface">
   <property name="text">
    <string>Interface</string>
   </property>
  </action>
  <action name="actionShotcuts">
   <property name="text">
    <string>Shotcuts</string>
   </property>
  </action>
  <action name="actionPlug_in">
   <property name="text">
    <string>Plug-Ins</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_2">
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionScreenshot">
   <property name="text">
    <string>Screenshot</string>
   </property>
  </action>
  <action name="actionFit_On_Screen">
   <property name="text">
    <string>Fit On Screen</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="text">
    <string>Fullscreen</string>
   </property>
  </action>
  <action name="actionAfficher_la_grille">
   <property name="text">
    <string>Show Grid</string>
   </property>
  </action>
  <action name="actionMagnetic_Grid">
   <property name="text">
    <string>Magnetic Grid</string>
   </property>
  </action>
  <action name="actionActualize">
   <property name="text">
    <string>Actualize</string>
   </property>
  </action>
  <action name="actionSelf_organization">
   <property name="text">
    <string>Self-organization</string>
   </property>
  </action>
  <action name="actionClass">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/icons/ToolClass.png</normaloff>:/icons/icons/ToolClass.png</iconset>
   </property>
   <property name="text">
    <string>Class</string>
   </property>
   <property name="toolTip">
    <string>Class</string>
   </property>
  </action>
  <action name="actionStruct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/icons/ToolStruct.png</normaloff>:/icons/icons/ToolStruct.png</iconset>
   </property>
   <property name="text">
    <string>Structure</string>
   </property>
   <property name="toolTip">
    <string>Structure</string>
   </property>
  </action>
  <action name="actionUnion">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/icons/ToolUnion.png</normaloff>:/icons/icons/ToolUnion.png</iconset>
   </property>
   <property name="text">
    <string>union</string>
   </property>
  </action>
  <action name="actionEnum">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/icons/ToolEnum.png</normaloff>:/icons/icons/ToolEnum.png</iconset>
   </property>
   <property name="text">
    <string>Enum</string>
   </property>
   <property name="toolTip">
    <string>Enum</string>
   </property>
  </action>
  <action name="actionNamespace">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/icons/ToolNamespace.png</normaloff>:/icons/icons/ToolNamespace.png</iconset>
   </property>
   <property name="text">
    <string>Namespace</string>
   </property>
   <property name="toolTip">
    <string>Namespace</string>
   </property>
  </action>
  <action name="action100">
   <property name="text">
    <string>100 %</string>
   </property>
  </action>
  <action name="action85">
   <property name="text">
    <string>75 %</string>
   </property>
  </action>
  <action name="action50">
   <property name="text">
    <string>50 %</string>
   </property>
  </action>
  <action name="action125">
   <property name="text">
    <string>125 %</string>
   </property>
  </action>
  <action name="action150">
   <property name="text">
    <string>150 %</string>
   </property>
  </action>
  <action name="action25">
   <property name="text">
    <string>25%</string>
   </property>
  </action>
  <action name="action175">
   <property name="text">
    <string>175 %</string>
   </property>
  </action>
  <action name="action200">
   <property name="text">
    <string>200 %</string>
   </property>
  </action>
  <action name="actionPublic">
   <property name="text">
    <string>public</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionProtected">
   <property name="text">
    <string>protected</string>
   </property>
  </action>
  <action name="actionPrivate">
   <property name="text">
    <string>private</string>
   </property>
  </action>
  <action name="actionComposition">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/icons/ToolComposition.png</normaloff>:/icons/icons/ToolComposition.png</iconset>
   </property>
   <property name="text">
    <string>Composition</string>
   </property>
  </action>
  <action name="actionAggregation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/icons/ToolAggregation.png</normaloff>:/icons/icons/ToolAggregation.png</iconset>
   </property>
   <property name="text">
    <string>Aggregation</string>
   </property>
   <property name="toolTip">
    <string>Aggregation</string>
   </property>
  </action>
  <action name="actionDependency">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/icons/ToolDependency.png</normaloff>:/icons/icons/ToolDependency.png</iconset>
   </property>
   <property name="text">
    <string>Dependency</string>
   </property>
  </action>
  <action name="actionFreeFunction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/icons/ToolFunction.png</normaloff>:/icons/icons/ToolFunction.png</iconset>
   </property>
   <property name="text">
    <string>Free Function</string>
   </property>
  </action>
  <action name="actionTemplate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/icons/ToolTemplate.png</normaloff>:/icons/icons/ToolTemplate.png</iconset>
   </property>
   <property name="text">
    <string>template</string>
   </property>
  </action>
  <action name="actionTypedef">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/icons/ToolTypedef.png</normaloff>:/icons/icons/ToolTypedef.png</iconset>
   </property>
   <property name="text">
    <string>typedef / template alias</string>
   </property>
  </action>
  <action name="actionAutocomplete">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Autocomplete</string>
   </property>
  </action>
  <action name="actionLibrary">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/icons/ToolLibrary.png</normaloff>:/icons/icons/ToolLibrary.png</iconset>
   </property>
   <property name="text">
    <string>Library</string>
   </property>
  </action>
  <action name="actionTools">
   <property name="text">
    <string>Tools</string>
   </property>
  </action>
  <action name="actionHistory">
   <property name="text">
    <string>History</string>
   </property>
  </action>
  <action name="actionFiles">
   <property name="text">
    <string>Files</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>C++ML Help</string>
   </property>
  </action>
  <action name="actionAbout_C_Ml">
   <property name="text">
    <string>About C++ML</string>
   </property>
  </action>
  <action name="actionC_Website">
   <property name="text">
    <string>C++ Website</string>
   </property>
  </action>
  <action name="actionCursor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/icons/ToolCursor.png</normaloff>:/icons/icons/ToolCursor.png</iconset>
   </property>
   <property name="text">
    <string>Cursor</string>
   </property>
   <property name="toolTip">
    <string>Cursor</string>
   </property>
  </action>
  <action name="actionInheritance">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/icons/ToolInheritance.png</normaloff>:/icons/icons/ToolInheritance.png</iconset>
   </property>
   <property name="text">
    <string>Inheritance</string>
   </property>
   <property name="toolTip">
    <string>Inheritance</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
